pyccel/errors/messages.py

```diff
@@ -123,6 +123,10 @@
 INVALID_FILE_EXTENSION = 'Wrong file extension. Expecting `py` of `pyh`, but found'
 INVALID_PYTHON_SYNTAX = 'Python syntax error'
 
+# Assignment errors
+REASSIGN_TARGET = 'can not reassign a target variable'
+ASSIGN_ALLOCATABLES = 'can not assign two allocatable variables'
```

Comment:
If possible we  should avoid using words like `target` and `allocatable` in the error messages. These are phrases that we use internally which come from fortran, but there is no reason to expect that someone translating from python to c would understand what is meant by them and even less reason to expect that a basic python user (who does not usually look at the generated code) would understand them

Reply:
i will try to make the error msgs more clear